controllers/commontestutils/testUtils.go

```diff
@@ -141,6 +142,27 @@ func NewOtherNodePlacement() *sdkapi.NodePlacement {
 	return getNodePlacement(3, 4)
 }
 
+func ExpectedPodAntiAffinity(appComponent hcoutil.AppComponent) *corev1.Affinity {
+	return &corev1.Affinity{
+		PodAntiAffinity: &corev1.PodAntiAffinity{
+			RequiredDuringSchedulingIgnoredDuringExecution: []corev1.PodAffinityTerm{
+				{
+					LabelSelector: &metav1.LabelSelector{
+						MatchExpressions: []metav1.LabelSelectorRequirement{
+							{
+								Key:      hcoutil.AppLabelComponent,
+								Operator: metav1.LabelSelectorOpIn,
+								Values:   []string{string(appComponent)},
+							},
+						},
+					},
+					TopologyKey: corev1.LabelHostname,
+				},
+			},
+		},
+	}
+}
+
```

Comment:
oh, this is the function I really want to comment about. sorry, these two functions share similar visual (:

So I would move **this one** to the file where it actually used, and make it also not exported.

Reply:
makes sense.
the function has been moved :+1: